compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/types/ParameterizedTypeTest.java

```diff
@@ -91,8 +91,8 @@ public void testInValidModuleLevelMapTypeWithInvalidTokenInside() {
         test("map<%> a;", "parameterized-type/parameterized_type_assert_10.json");
     }
 
-    @Test
-    public void testInValidModuleLevelParameterizedTypeMissingType() {
-        test("<int> a;", "parameterized-type/parameterized_type_assert_12.json");
-    }
+//    @Test
+//    public void testInValidModuleLevelParameterizedTypeMissingType() {
+//        test("<int> a;", "parameterized-type/parameterized_type_assert_12.json");
+//    }
```

Comment:
This test is not needed?

Reply:
`<int> a;` is recovering as `error<int> a;` so I commented the parameterized test case.